extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -516,11 +515,14 @@ export abstract class PostgresParametersPage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
-							const session = await this._postgresModel.controllerModel.acquireAzdataSession();
+							let session: azdataExt.AzdataSession | undefined = undefined;
 							try {
-								this.resetParameter(engineSetting.parameterName!, session);
+								session = await this._postgresModel.controllerModel.acquireAzdataSession();
+								await this.resetParameter(engineSetting.parameterName!, session);
+							} catch (err) {
+								throw err;
```

Comment:
This is unnecessary - if you don't have a catch it'll still throw (but first do the finally)